lib/Cargo.toml

```diff
@@ -42,7 +42,12 @@ nom = { version = "^7.1.3", default-features = true, features = ["std"] }
 poule = "^0.3.2"
 rand = "^0.8.5"
 regex = "^1.11.0"
-rustls = { version = "^0.23.14", features = ["ring"] }
+rustls = { version = "^0.23.14", default-features = false, features = [
```

Comment:
There was a dependency error during docker build caused by an AWS related feature thats in defaults in rustls i just removed it